erpnext_com/www/docs/user/manual/en/setting-up/integrations/ldap-integration.md

```diff
@@ -1,35 +1,103 @@
 <!-- add-breadcrumbs -->
```

Comment:
this belongs to /docs/user/manual/en/erpnext_integration
A file already exists there. perhaps merge them.

Reply:
Hi @prasadarr  that's what I did  - sorry I'm a bit confused by your comment?

Reply:
Hi @cjpit, integrations had a duplicate path in setting-up and i fixed it recently, so integrations will not appear under /en/setting-up/
integration files need to be put under /docs/user/manual/en/erpnext_integration
a git pull will show you the updates